### ISSUE TITLE
gitg: SHA256 mismatch

### ISSUE THREAD

**Issue by valentinegb** [Homebrew/homebrew-core#173703]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
https://gist.github.com/valentinegb/d452b9d90f5c04180db8e76580f5a75f
```

### Verification

- [X] My `brew doctor` output says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

I was trying to install the `gitg` formula with the flag `--build-from-source` to prove that it no longer builds on the `master` branch and that this failure is not related to #173550.

### What happened (include all command output)?

```
==> Downloading https://formulae.brew.sh/api/formula.jws.json
##O=- #     #
==> Downloading https://formulae.brew.sh/api/cask.jws.json

==> Fetching dependencies for gitg: harfbuzz, gpgme, gtk-mac-integration, gspell, gtksourceview4, libdazzle, libssh2, libgit2, libgit2-glib, libhandy and intltool
==> Fetching harfbuzz
==> Downloading https://ghcr.io/v2/homebrew/core/harfbuzz/manifests/8.5.0
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/3a4d22064fd474a301c7c45c5856ce7ad0d93b8ec485ec842972eab8ede343d3--harfbuzz-8.5.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/harfbuzz/blobs/sha256:27ef5184c6701225923ed823c9f142cb4c3da654c9c0411228694d022d3f9958
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/0ac2a6fccb042eeca7e4595ba7cdc37f12cf18d9e0f2573f73445d66381bc2db--harfbuzz--8.5.0.arm64_sonoma.bottle.tar.gz
==> Fetching gpgme
==> Downloading https://ghcr.io/v2/homebrew/core/gpgme/manifests/1.23.2_1
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/d2fe23e81c046d445218bb6231927431d4d7dff0d7bc727cf237511500fcbf59--gpgme-1.23.2_1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/gpgme/blobs/sha256:eedb9535788809e98ab8feebae75fabd5ffa39f89ec5fcc50b5d7b36ad612e69
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/de6005992e2d502be054d0715b85cb7e38507559ae314a8860098e4d0b08a9f5--gpgme--1.23.2_1.arm64_sonoma.bottle.tar.gz
==> Fetching gtk-mac-integration
==> Downloading https://ghcr.io/v2/homebrew/core/gtk-mac-integration/manifests/3.0.1_1-1
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/48b2a7a5e9dde291c7113e117d82da2342ef31c2a875daf75e07feb8206b72fc--gtk-mac-integration-3.0.1_1-1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/gtk-mac-integration/blobs/sha256:b538f4f624bda8680cf08cd1a26a41f3dd8e0d16c4c52db4543a737554c068a2
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/27fca0a56a39ca44af73b2b1b98e951cf1605badc9030613a47cfec2ae613801--gtk-mac-integration--3.0.1_1.arm64_sonoma.bottle.1.tar.gz
==> Fetching gspell
==> Downloading https://ghcr.io/v2/homebrew/core/gspell/manifests/1.12.2_2
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/02116d7acb0c61d29eaa99e3175146e5ca6de736fcc3f63fd68fa6d0df9c045d--gspell-1.12.2_2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/gspell/blobs/sha256:e3c60c553bc207bd79782ff8057f1a2a100ae83abdea680e07a80890d409de6f
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/66b643ca3c82aac5a0560267a6dc1c9393386d97bbe6a77962ee17eea0aa12d7--gspell--1.12.2_2.arm64_sonoma.bottle.tar.gz
==> Fetching gtksourceview4
==> Downloading https://ghcr.io/v2/homebrew/core/gtksourceview4/manifests/4.8.4_1
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/cb52ba01506a81483bd35ae39db1bea47f8de3eeb14e515d880f776b2b032d08--gtksourceview4-4.8.4_1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/gtksourceview4/blobs/sha256:b07d4c344cf2681c5b78e3ee653a8d2933751508b6378707532e570dc2afa3f9
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/e828d3213755e35a45fef818e6cb365680a90bf6bd4282bbe626976128c5a690--gtksourceview4--4.8.4_1.arm64_sonoma.bottle.tar.gz
==> Fetching libdazzle
==> Downloading https://ghcr.io/v2/homebrew/core/libdazzle/manifests/3.44.0_1
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/63ac663c4fff65b098caa07dba5e541ea1d641c3b6e2dc1b98b9f61bfb9344b3--libdazzle-3.44.0_1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/libdazzle/blobs/sha256:f79987d1cba7d798fe6cb2c824439ddb41dbac57df64c75dc8cf72e66a9c0d25
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/a80a8669986f1d29226e3de883e83a1720a68d8a0fc86f8cfa73c293b994fb10--libdazzle--3.44.0_1.arm64_sonoma.bottle.tar.gz
==> Fetching libssh2
==> Downloading https://ghcr.io/v2/homebrew/core/libssh2/manifests/1.11.0_1
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/48ca0c7785b21630a4817c59b72205609ccb0575e7abc64d64af2e61a60b5b0a--libssh2-1.11.0_1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/libssh2/blobs/sha256:ec12598827b91ddee213c24b17a2bba833e5e1cbcfaf25b0d514dfd3fe03f527
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/a70a2917adfe875f9751c7e6991318d8e941194eda0897f96cec4d067a47c44a--libssh2--1.11.0_1.arm64_sonoma.bottle.tar.gz
==> Fetching libgit2
==> Downloading https://ghcr.io/v2/homebrew/core/libgit2/manifests/1.7.2
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/9671856ced457ee95383408b4c243c7f196b6f204228b11a53735a8a561e2abb--libgit2-1.7.2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/libgit2/blobs/sha256:20b0ffc207c455904322e9b34ab8bb0754673009f903f3e2e5e72624e5c8d434
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/7254f533f42b790c136a37d54e65a1cfe5d41ba61fb4210983bbff3e2be71134--libgit2--1.7.2.arm64_sonoma.bottle.tar.gz
==> Fetching libgit2-glib
==> Downloading https://ghcr.io/v2/homebrew/core/libgit2-glib/manifests/1.2.0
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/ec983457aaea72752fae89d51499f4b1770c4d7e30cbf79345c951e0a2ef1f1e--libgit2-glib-1.2.0.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/libgit2-glib/blobs/sha256:bfcbc9f8a8bdd07cd9a5c62da9e50bc454d73269db42a7ba8f93c30eea2be410
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/1f4cdd7096417f9a4d54c3fe526673757e5a69a57207a09da43dd6b538d7b611--libgit2-glib--1.2.0.arm64_sonoma.bottle.tar.gz
==> Fetching libhandy
==> Downloading https://ghcr.io/v2/homebrew/core/libhandy/manifests/1.8.3
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/2c032a4257212824333b4052fe79c872e012c2eefc8dddfc22d25bf6b7694fa6--libhandy-1.8.3.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/libhandy/blobs/sha256:f1b399070cea7b5c44e3afad605155f867659920d5d3de17c466cf5cc3ba6a33
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/545a63d7910f0a4c456c72f1c1ab23fb1c50ef0bde38c06a8e5ac4ada93d0b2f--libhandy--1.8.3.arm64_sonoma.bottle.tar.gz
==> Fetching intltool
==> Downloading https://ghcr.io/v2/homebrew/core/intltool/manifests/0.51.0_2
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/ddd54ff3a016fe32ced2591ec33eac29be1fedf41eadde7d0a6d90f802d1e829--intltool-0.51.0_2.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/intltool/blobs/sha256:825910e5988270e48c5815bf1775a487d7b480c75a3128c1e800757a291119bb
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/e38270898e511e3fc5850ffd9c975aff1debc5a35c97b8e591389cf445e16477--intltool--0.51.0_2.arm64_sonoma.bottle.tar.gz
==> Fetching gitg
==> Downloading https://raw.githubusercontent.com/Homebrew/homebrew-core/64a641caa97cdf8c98425b0286e437ba13413828/Formula/g/gitg.rb
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/f20586bd622d1b0d81700101cf8b517079d79e8ff6b4fd672a359208d531dc64--gitg.rb
==> Downloading https://download.gnome.org/sources/gitg/44/gitg-44.tar.xz
Already downloaded: /Users/valentinebriese/Library/Caches/Homebrew/downloads/bc9599dec341974699ec17a60689e44d1c9ade7f7a6ca727987be6dbe6d91f65--gitg-44.tar.xz
Error: gitg: SHA256 mismatch
Expected: 5b0e99ab3e7b94b0daa98ca8041d5ec9280ee0a2c28338a5506a968ac52e2354
  Actual: 342a31684dab9671cd341bd3e3ce665adcee0460c2a081ddc493cdbc03132530
    File: /Users/valentinebriese/Library/Caches/Homebrew/downloads/bc9599dec341974699ec17a60689e44d1c9ade7f7a6ca727987be6dbe6d91f65--gitg-44.tar.xz
To retry an incomplete download, remove the file above.
```

### What did you expect to happen?

Well really I was expecting this to happen because it happened in the CI for #173550, but it *should* have built and installed without error.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install --build-from-source gitg
```

**Comment by valentinegb** [comment]
This upstream issue regarding the SHA256 change was created while investigating audit failures in #173550: https://gitlab.gnome.org/GNOME/gitg/-/issues/464

**Comment by valentinegb** [comment]
A developer said that the version was rereleased to fix the version number being incorrectly defined, which checks out since the `inreplace` to fix the version in the formula no longer works.

**Comment by branchvincent** [comment]
@valentinegb thank you for confirming the change upstream! fixed in #173728
